maui-toolkit/Bottom-Sheet/BottomSheet-Customization.md

```diff
@@ -67,11 +67,11 @@ Content = bottomSheet;
 {% endtabs %}
 
 ## Allowed State
-The states of the `BottomSheet` can be controlled using the `AllowedState` property. The default value is `All`, allowing all states. This property accepts the following values:
+The states of the `BottomSheet` allow you to control the height of the bottom sheet using the `AllowedState` property. The default value is `All`, which allows transitions between all available states. This property accepts the following values:
 
-*   **FullExpanded** - The sheet can expand to cover the full screen.
-*   **HalfExpanded** - The sheet can expand to half of the screen height.
-*   **All** - The sheet can transition between all available states (FullExpanded, HalfExpanded, Collapsed, and Hidden).
+* **FullExpanded** - Restricts the transitions to only between FullExpanded and Collapsed states.
```

Comment:
It will allows the transition.

Reply:
updated the FullExpanded and HalfExpanded sentance.